### PR TITLE
Add Fade To Carousel Edges

### PR DESCRIPTION
**Pull request by fcescob**
Solves #771 , review after #1562 is merged

### CODE REVIEW

**Comment by kdembler** [comment]
Try clicking some element in the carousel :P

**Review by kdembler — COMMENTED** [review]
Much better now :P Shouldn't there be a fade for the left arrow as well when it appears though?

File: atlas/src/shared/components/Button/Button.tsx [hunk 1]
```diff
@@ -11,40 +11,48 @@ export type ButtonProps = {
...
-  containerCss,
-  className,
-  onClick,
-}) => {
-  const clickable = !!onClick
-  const hasText = !!children
-  const handleClick = (e: React.MouseEvent<HTMLButtonElement>) => {
-    if (!onClick) return
-    onClick(e)
+const Button = React.forwardRef<HTMLButtonElement, Partial<ButtonProps>>(
```
Comment (kdembler) [inline atlas/src/shared/components/Button/Button.tsx]: I think it'd be best to separate component declaration and `React.forwardRef`, for readability

**Review by kdembler — COMMENTED** [review]
To summarize the points from the call:
- let's remove the old `Glider` component
- let's try to abstract some Glider-specific logic outside of `Carousel` component (like adding event handlers and so on)
- let's use pure CSS for adding the shadows so that we don't need any kind of scroll event handler

**Comment by fcescob** [comment]
I believe #1709 is fixed too in this PR

**Review by kdembler — APPROVED** [review]
LGTM!

**Review by bedeho — APPROVED** [review]

**Review by bedeho — APPROVED** [review]

**Pull request merged**